File: app/src/main/java/com/elseboot3909/GCRClient/Entities/CommitInfo.java
package com.elseboot3909.GCRClient.Entities;

public class CommitInfo {
    private String commit;
    private CommitInfo parent;
    private GitPersonInfo author;
    private GitPersonInfo committer;
    private String subject;
    private String message;

    public String getMessage() { return message; }
}

File: app/src/main/java/com/elseboot3909/GCRClient/Entities/CommitInfo.kt
package com.elseboot3909.GCRClient.Entities

import java.io.Serializable

data class CommitInfo (
        val commit: String = "",
        val author: GitPersonInfo = GitPersonInfo(),
        val committer: GitPersonInfo = GitPersonInfo(),
        val subject: String = "",
        val message: String = ""
)  : Serializable
